src/setuptools_scm/config.py

```diff
@@ -82,6 +84,8 @@ def __init__(
         self.tag_regex = tag_regex
         self.git_describe_command = git_describe_command
         self.dist_name = dist_name
+        self.version_cls = version_cls
+        self.normalize = normalize
```

Comment:
this should integrate with version_cls in the way that when setting it to true, it should choose the NonNormalized version class and/or assert its either already being used or its not replacing a different choice